app/models/data/returning_users.rb

```diff
@@ -5,22 +5,16 @@ class ReturningUsers
     class << self
       # @return [Array<String>]
       def column_names
-        [
-          'Weekly Returning Users',
-          'Monthly Returning Users',
-          'Bimonthly Returning Users',
-          'Quarterly Returning Users',
-        ]
+        ['Weekly Returning Users', 'Monthly Returning Users', 'Quarterly Returning Users']
       end
 
       # @return [Array<Hash{Symbol => Mixed}>]
       def dashboard
         [
           {
-            weekly: weekly,
-            monthly: monthly,
-            bimonthly: bimonthly,
-            quarterly: quarterly,
+            weekly_returning_users: weekly,
```

Comment:
My rationale for renaming the keys was brevity. The `_returning_user` suffix gives me more to read.